tox.ini

```diff
@@ -3,7 +3,8 @@ skipsdist=True
 envlist=
     lint,
     python{3.8}-django{3.2}-wagtail{3.0},
-    python{3.8,3.11}-django{3.2,4.1}-wagtail{4.1,4.2},
+    python{3.11}-django{4.1}-wagtail{4.1,4.2,5.0,5.1},
+    python{3.11}-django{4.2}-wagtail{5.0,5.1},
```

Comment:
Might be good to replace this with our current standard envlist:
```
python{3.8,3.9,3.10}-django{3.2,4.1}-wagtail{4.1,4.2,5.0,5.1}
python3.11-django4.1-wagtail{4.1,4.2,5.0,5.1}
python3.11-django4.2-wagtail{5.0,5.1}
```

Reply:
I wanted to keep the maintainers testing for v3.0 in place so have expanded the last line to just add the new versions. I think the maintainer would like to see that for now, we can always adjust it if they want to do that.